### DISCUSSION TITLE
What URL does the $.ajax's need?

### DISCUSSION THREAD

**Discussion by justinjalandoniiwest** [jquery/jquery#4952] [Q&A]
In this `$.ajax` code:

```js
$.ajax({
    url: "whatUrlToPlaceHere",
    type: "Post",
    success: function (result) {
        alert(result);
    },
    error: function (jqXHR, textStatus, errorThrown) {
        console.log(jqXHR);
        console.log(textStatus);
        console.log(errorThrown);
    },
});
```

What value should we place in the `url` property? Is it the absolute path of the URL we need? What if the url is deployed locally?

**Comment by mgol** [comment]
It can be the full URL, it can just be the path part, there are multiple options so use what is the easiest for you. It doesn't matter if you're running it on a local dev server or not.
